pkg/backingstore/reconciler.go

```diff
@@ -30,30 +30,35 @@ import (
-	return map[string]nbv1.BackingStorePhaseInfo{
-		"INITIALIZING":        {nbv1.BackingStorePhaseReady, "BackingStorePhaseReady", "Backing store mode: INITALIZING"},
-		"DELETING":            {nbv1.BackingStorePhaseReady, "BackingStorePhaseReady", "Backing store mode: DELETING"},
-		"SCALING":             {nbv1.BackingStorePhaseReady, "BackingStorePhaseReady", "Backing store mode: SCALING"},
-		"MOST_NODES_ISSUES":   {nbv1.BackingStorePhaseReady, "BackingStorePhaseReady", "Backing store mode: MOST_NODES_ISSUES"},
-		"MANY_NODES_ISSUES":   {nbv1.BackingStorePhaseReady, "BackingStorePhaseReady", "Backing store mode: MANY_NODES_ISSUES"},
-		"MOST_STORAGE_ISSUES": {nbv1.BackingStorePhaseReady, "BackingStorePhaseReady", "Backing store mode: MOST_STORAGE_ISSUES"},
-		"MANY_STORAGE_ISSUES": {nbv1.BackingStorePhaseReady, "BackingStorePhaseReady", "Backing store mode: MANY_STORAGE_ISSUES"},
-		"MANY_NODES_OFFLINE":  {nbv1.BackingStorePhaseReady, "BackingStorePhaseReady", "Backing store mode: MANY_NODES_OFFLINE"},
-		"LOW_CAPACITY":        {nbv1.BackingStorePhaseReady, "BackingStorePhaseReady", "Backing store mode: LOW_CAPACITY"},
-		"OPTIMAL":             {nbv1.BackingStorePhaseReady, "BackingStorePhaseReady", "Backing store mode: OPTIMAL"},
-		"HAS_NO_NODES":        {nbv1.BackingStorePhaseRejected, "BackingStorePhaseRejected", "Backing store mode: HAS_NO_NODES"},
-		"ALL_NODES_OFFLINE":   {nbv1.BackingStorePhaseRejected, "BackingStorePhaseRejected", "Backing store mode: ALL_NODES_OFFLINE"},
-		"NO_CAPACITY":         {nbv1.BackingStorePhaseRejected, "BackingStorePhaseRejected", "Backing store mode: NO_CAPACITY"},
-		"IO_ERRORS":           {nbv1.BackingStorePhaseRejected, "BackingStorePhaseRejected", "Backing store mode: IO_ERRORS"},
-		"STORAGE_NOT_EXIST":   {nbv1.BackingStorePhaseRejected, "BackingStorePhaseRejected", "Backing store mode: STORAGE_NOT_EXIST"},
-		"AUTH_FAILED":         {nbv1.BackingStorePhaseRejected, "BackingStorePhaseRejected", "Backing store mode: AUTH_FAILED"},
+func modeInfoMap() map[string]ModeInfo {
+	return map[string]ModeInfo{
+		"INITIALIZING":        {nbv1.BackingStorePhaseReady, corev1.EventTypeNormal},
+		"DELETING":            {nbv1.BackingStorePhaseReady, corev1.EventTypeNormal},
+		"SCALING":             {nbv1.BackingStorePhaseReady, corev1.EventTypeNormal},
+		"MOST_NODES_ISSUES":   {nbv1.BackingStorePhaseReady, corev1.EventTypeNormal},
+		"MANY_NODES_ISSUES":   {nbv1.BackingStorePhaseReady, corev1.EventTypeNormal},
+		"MOST_STORAGE_ISSUES": {nbv1.BackingStorePhaseReady, corev1.EventTypeNormal},
+		"MANY_STORAGE_ISSUES": {nbv1.BackingStorePhaseReady, corev1.EventTypeNormal},
+		"MANY_NODES_OFFLINE":  {nbv1.BackingStorePhaseReady, corev1.EventTypeNormal},
+		"LOW_CAPACITY":        {nbv1.BackingStorePhaseReady, corev1.EventTypeNormal},
+		"OPTIMAL":             {nbv1.BackingStorePhaseReady, corev1.EventTypeNormal},
+		"HAS_NO_NODES":        {nbv1.BackingStorePhaseRejected, corev1.EventTypeNormal},
```

Comment:
why all of them are normal? didn't we want to change it for some to be warnings and some normal?

Reply:
Yed I saw it after the push, and already pushed a fix.
Thanks